lib/loader/serialize/serial_type.cpp

```diff
@@ -55,6 +69,15 @@ Serializer::serializeRefType(const ValType &Type, ASTNodeAttr From,
       OutVec.push_back(static_cast<uint8_t>(Code));
     }
     return serializeHeapType(Type, From, OutVec);
+  case TypeCode::NullFuncRef:

```

Comment:
Not need the following cases.
Because in the `class ValType`, we migrated the ref-related cases into the `ref` or `ref.null`, and handled in the heap type.

Reply:
Not fixed.